collections/_patterns/email-address.html

```diff
@@ -0,0 +1,71 @@
+---
+title: Email Address
+description: Use this pattern when you need to ask users for their email address.
+react-link: https://1-0-0-dev.dg2jutx9gylat.amplifyapp.com/?path=/docs/components-form-patterns--email#email-pattern
+github-link: true
+toc: true
+---
+
+{% capture email-pattern %}
+<div class="sgds form-group">
+    <label class="form-label" for="email">Email Address</label>
+    <span class="form-text">Your email will only be used for updates</span>
+    <input class="form-control" type="email" id="email" autocomplete="email" spellcheck="false" placeholder="Enter email" required />
+</div>
```

Comment:
need the validations too. can help add it in. can reference react's , got the invalid css selectors . if need be add javascript